### PR TITLE
Seeds CSV file of 10MM records for agnostic database insert.

### PR DESCRIPTION
**Pull request by jayrbarr**

### CODE REVIEW

File: database/models.js [hunk 1]
```diff
@@ -99,17 +99,53 @@ const Game = mongoose.model('Game', gameSchema);
...
+const changeOne = ({ id, body }, callback) => {
+  const filter = { proxyId: id };
+  Game.updateOne(filter, body, (err) => {
+    if (err) {
+      callback(err);
+    } else {
+      console.log('changeOne success');
+      callback();
+    }
+  });
```
Comment (JasperCDev) [inline database/models.js]: is this for the put request?

File: database/seedcsv.js [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+      faker.date.past()},${faker.image.image()},${reviewRatings[Math.floor(Math.random() * reviewRatings.length)]},${
+      Math.floor(Math.random() * 1001)},\'${JSON.stringify([...new Set(xTags)])}\',${
+      faker.commerce.productName()},${faker.image.image()},${faker.commerce.price()},${
+      faker.date.past()},${faker.image.image()},${reviewRatings[Math.floor(Math.random() * reviewRatings.length)]},${
+      Math.floor(Math.random() * 1001)},\'${JSON.stringify([...new Set(xTags)])}\',${
+      faker.commerce.productName()},${faker.image.image()},${faker.commerce.price()},${
+      faker.date.past()},${faker.image.image()},${reviewRatings[Math.floor(Math.random() * reviewRatings.length)]},${
+      Math.floor(Math.random() * 1001)},\'${JSON.stringify([...new Set(xTags)])}\'\n`;
+    try {
+      fs.appendFileSync('data.csv', data);
```
Comment (JasperCDev) [inline database/seedcsv.js:211]: how were you able to do this without creating a stream?

**Review by JasperCDev — APPROVED** [review]
lgtm

**Review by timothydadams — APPROVED** [review]
Looks good!

File: database/models.js (hunk 1 above)
Comment (jayrbarr) [reply database/models.js]: Yes.

File: database/seedcsv.js (hunk 2 above)
Comment (jayrbarr) [reply database/seedcsv.js:211]: It's a synchronous command. Just executes the write on one row at a time and waits. Slow but it gets the job done. I want to convert to the stream though.

Comment (JasperCDev) [reply database/seedcsv.js:211]: ah I see, thank you Jay!

**Pull request merged**